ts/private/ts_lib.bzl

```diff
@@ -29,6 +81,14 @@ See more details on the `assets` parameter of the `ts_project` macro.
     "out_dir": attr.string(
         doc = "https://www.typescriptlang.org/tsconfig#outDir",
     ),
+    "resource_set": attr.string(
```

Comment:
I feel like if we're using the bazel `resource_set` term we should be using the bazel `resource_set` API and not create our own enum API. How will this current API expand in the future to support changing the CPU attribute?

Maybe just rename this to `resource_set_memory`? Or use the bazel `resource_set` API?

Reply:
updated to use the feature that landed in bazel-lib 2.x